src/components/feilmelding/Feilmelding.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { STATUS } from '../../redux/utils';
+import { AlertStripeAdvarsel } from 'nav-frontend-alertstriper';
+import { Reducer } from '../../redux/reducer';
+
+const FeilmeldingWrapper = styled.div`
+  margin: 15px;
+`;
+
+interface FeilmeldingProps {
+    reducer: Reducer<object>;
+}
+
+function Feilmelding({reducer}: FeilmeldingProps) {
+    if (reducer.status === STATUS.ERROR) {
+        return (
+            <FeilmeldingWrapper>
```

Comment:
Hvorfor trengs denne? Kall den gjerne noe som sier hva den gjør hvis mulig, f.eks "LittMargin" eller "Midtstill"

Reply:
Den trengs for å gi litt luft rundt feilmelding. Kan endre navn, men først skitur 🌞